apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.IntegrationTests/JobsTest.cs

```diff
@@ -44,11 +46,200 @@ public void ListJobs()
             var oneMinuteAgo = ToMillisecondsSinceEpoch(DateTime.UtcNow.AddMinutes(-1));
 
             // Find all jobs started in the last minute.
-            var recentJobs = client.ListJobs().TakeWhile(job => job.Statistics.CreationTime >= jobToFind.Statistics.CreationTime).ToList();
+            var recentJobs = client.ListJobs().TakeWhile(job => job.Statistics.CreationTime >= oneMinuteAgo).ToList();
```

Comment:
Hmm. I'm surprised the compiler didn't point out that I wasn't using the variable. Thanks for that!

And yes, we are relying on the ordering, but it's documented:

> The job list is sorted in reverse chronological order, by job creation time. 

It wouldn't hurt to add that to a comment though :)

Reply:
Adding comment on the use of TakeWhile.

I think the compiler is not set to catch unused variables, for instance, in this same test the variable table wasn't being used either because the $ was missing from the front of the query string.